pydicom/values.py

```diff
@@ -156,12 +164,24 @@ def convert_DS_string(byte_string, is_little_endian, struct_format=None):
         ``DSdecimal``, otherwise returns :class:`~pydicom.valuerep.DSfloat` or
```

Comment:
The Returns section of the docstring needs to be updated

Reply:
Didn't notice this comment ... will update to address the return docs and push again.

Reply:
Got something working, but only if use very long line length for Return type  -- help appreciated to break the line without breaking the sphinx formatting.

Reply:
That turned out pretty well